apps/backoffice-v2/src/pages/Entities/components/CaseGeneration/components/CaseGenerationForm/hooks/useFormSchema/useFormSchema.ts

```diff
@@ -0,0 +1,26 @@
+import { TWorkflowDefinition } from '@/domains/workflows/fetchers';
+import { useMemo } from 'react';
+
+export const useFormSchema = (workflowDefinition: TWorkflowDefinition) => {
+  const transitionSchema = useMemo(
+    () =>
+      workflowDefinition?.transitionSchema?.find(
+        schema => schema.state === workflowDefinition?.definition?.initial,
+      ) ?? null,
```

Comment:
Prefer `undefined` to `null` unless an is an exception. `null` from React Query and Prisma are just instances out of our control.
```suggestion
      ),
```